doc/mgr/administrator.rst

```diff
@@ -44,12 +44,12 @@ Calling module commands
 Where a module implements command line hooks, using the Ceph CLI's
 ``tell`` command to call them like this::
 
-    ceph tell mgr <command | help>
+    ceph tell mgr <command | get_command_descriptions>
```

Comment:
i am not very sure if the json output of `get_command_description` is supposed to be consumed by human being...